pallets/gear-voucher/src/internal.rs

```diff
@@ -144,6 +145,7 @@ pub trait PrepaidCallsDispatcher {
     fn dispatch(
         account_id: Self::AccountId,
         sponsor_id: Self::AccountId,
+        voucher_id: VoucherId,
```

Comment:
`sponsor_id` is now exceeded and must be gotten inside `dispatch `function for corresponding calls:
`let sponsor_id = voucher_id.cast()`

Reply:
deprecated vouchers can also use only `voucher_id`

Reply:
At the moment it makes sense, but future changes to vouchers will bring us back to 3 params because sponsor won't be eq voucher_id.cast(), so I'd keep it atm in favour of more clear signature and usage of dispatcher